eng/testing/linker/SupportFiles/Directory.Build.props

```diff
@@ -13,5 +12,8 @@
     <EnableTrimAnalyzer>false</EnableTrimAnalyzer>
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
     <NoWarn>$(NoWarn);IL2121</NoWarn>
+
+    <!-- EventSourceSupport is disabled by default with PublishAot=true, so need to enable it here. -->
+    <EventSourceSupport>true</EventSourceSupport>
```

Comment:
Does this enable EventSource for everything always? I've been adding it to only those tests that need it, like so:

https://github.com/dotnet/runtime/blob/d54a47d5fdd2b2c57a88f3e816a3c6d62fdf0fcd/src/libraries/Microsoft.Extensions.Logging.EventSource/tests/Microsoft.Extensions.Logging.EventSource.Tests.csproj#L6

Motivation:
1. Better to test what our customers will be running.
2. Less code to compile == faster CI